docker-compose.yml

```diff
@@ -28,3 +28,5 @@ services:
     volumes:
       - ${CACHE_ROOT}/cache:/data/cache
       - ${CACHE_ROOT}/logs:/data/logs
+    environment:
+      - TZ=${TZ}
```

Comment:
If we are already pulling in the .env, do we need the TZ=TZ here?

And if we do, why isn't it in dns as well?

Reply:
I don't see the point either why we would need TZ=$TZ here. The .env file alone should replace this.

Reply:
> If we are already pulling in the .env, do we need the TZ=TZ here?

You're right, we don't, I'll remove it from the compose file.
> And if we do, why isn't it in dns as well?

It was, but was lost in a merge with the newer dns. Doesn't matter now regardless, since I'll remove it from the compose file.

Fixed in https://github.com/lancachenet/docker-compose/pull/18/commits/cec39696f1c695b59373b91c07fd9f4a4cef2dc0